docs/machine-learning/ml-problem-formulation-envisioning.md

```diff
@@ -1,76 +1,92 @@
 
 The following questions can help guide discussion in understanding the stakeholders' perspectives:
 
-* Who is the end user?
-* What is the current practice related to the business problem?
-* What's the performance of the current solution?
-* What are their pain points?
-* What is their toughest problem?
-* What is the state of the data used to build the solution?
-* How does the end user or SME envision the solution?
-
-## Envisioning Guidance
-
-During envisioning sessions, the following may prove useful for guiding the discussion. Many of these points are taken directly, or adapted from, [[1]](#references) and [[2]](#references).
-
-### Problem Framing
-
-1. Define the objective in business terms.
-2. How will the solution be used?
-3. What are the current solutions/workarounds (if any)? What work has been done in this area so far? Does this solution need to fit into an existing system?
-4. How should performance be measured?
-5. Is the performance measure aligned with the business objective?
-6. What would be the minimum performance needed to reach the business objective?
-7. Are there any known constraints around non-functional requirements that would have to be taken into account? (e.g., computation times)
-8. Frame this problem (supervised/unsupervised, online/offline, etc.)
-9. Is human expertise available?
-10. How would you solve the problem manually?
-11. Are there any restrictions on the type of approaches which can be used? (e.g., does the solution need to be completely explainable?)
-12. List the assumptions you or others have made so far. Verify these assumptions if possible.
-13. Define some initial hypothesis statements to be explored.
-14. Highlight and discuss any responsible AI concerns if appropriate.
-
-### Workflow
+*	Who are the people affected by the outcomes of the proposed system? (These may not be just end users)
+*	What is the current practice related to the business problem?
+*	What are the local initiatives related to the current problem statement? How can we learn from, leverage, and build on those efforts?
+*	What is the current solution? How does it perform?
+*	What opportunities or untapped potential exists in the problem space? 
+*	What are the frictions experienced by people who are affected by the problem today?
+*	What are the most consistent or pressing challenges related to the problem?
+*	What is the state of the data used to build the solution?
+*	How does the end user or SME envision the solution?
+
+
+## Envisioning Phase
+
+During the envisioning phase and related efforts, the following guides may prove useful for directing the discussion and activities, as well as identifying and coordinating with key resources on the project. Many of these points are taken directly, or adapted from, [[1]](#references) and [[2]](#references).
+
+### Problem Framing & Hypothesis Generation
```

Comment:
Hypothesis generation seem to me like an activity that should be done at the next step in the project. Should it be part of the envisioning phase? We usually perform hypothesis driven analysis/development once we start data and business understanding (i.e., the problem is already formulated).

Reply:
Part of Discovery, which for HX Research is Descriptive and Generative research of the business and human context, would be "generating theory to test" e.g. hypothesis. It would not necessarily include the testing of those theories, but would at least provide them as part of the "findings" of a discovery.

Would love to collaborate on this and get more of your feedback in early December if you are around then.